products/mcp/schema/tool-definitions.json

```diff
@@ -125,6 +125,20 @@
             "readOnlyHint": true
         }
     },
+    "update-issue": {
+        "description": "Update an error tracking issue's status or name. Allows you to mark errors as resolved, suppressed, or change their status. You can also rename issues for better organization.",
+        "category": "Error tracking",
+        "feature": "error-tracking",
+        "summary": "Update an error tracking issue's status or name.",
```

Comment:
Should it be possible to update the assignee too? Feel free to skip if that's outside the scope of what you want to do